### DISCUSSION TITLE
Detect User Permission Multiple automatic file download on browser

### DISCUSSION THREAD

**Discussion by KamalEgon** [axios/axios#5883] [Q&A]
### Describe the issue

How to detect user has blocked multiple automatic file download on browser is there any method provided by the axios
<img width="400" alt="Screen Shot 2023-01-20 at 1 49 09 PM" src="https://user-images.githubusercontent.com/123153866/213649013-9e39209d-156b-48e3-85ca-ed4d0b2ac912.png">

### Example Code

```js
Looking for code or function to detect user has blocked multiple automatic file download on browser
```

### Expected behavior

Method to check if user has blocked multiple automatic file download on browser

### Axios Version

1.2.1

### Browser

Chrome

### Browser Version

109.0.5414.87 

### Additional Library Versions

```bash
react": "^18.2.0",
```

**Comment by stevenleep** [comment]
Maybe it's browser behavior

**Comment by jasonsaayman** [comment]
This is standard browser behaviour and outside of the scope of axios.
